Recast/Source/RecastLayers.cpp

```diff
@@ -27,7 +27,8 @@
 #include "RecastAssert.h"
 
 
-static const int RC_MAX_LAYERS = RC_NOT_CONNECTED;
+// Must be 255 or smaller
+static const int RC_MAX_LAYERS = 63;
```

Comment:
should leave comment here that max value is 254, as `0xff` is used in some places as "not set" (IIRC).